### PR TITLE
Add terser config to Serverless

### PR DESCRIPTION
**Pull request by Fry98**
## PR Type

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] Docs
- [ ] Other... Please describe:

## Current behavior

Issue Number: https://github.com/nestjs/nest/issues/8639

Without additional config changes to `terser-webpack-plugin`, the production build for serverless doesn't work properly.

## New behavior

This PR adds the necessary configuration to the docs.

## Does this PR introduce a breaking change?
- [ ] Yes
- [x] No

### CODE REVIEW

**Review by micalevisk — CHANGES_REQUESTED** [review]
can you revert the changes made on `src/app`? Just need to touch `content` directory

**Comment by Fry98** [comment]
> can you revert the changes made on `src/app`? Just need to touch `content` directory

Ah, sorry about that. Fixed it.

File: content/faq/serverless.md [hunk 1]
```diff
@@ -263,13 +263,22 @@ $ npx serverless offline
 Once the application is running, open your browser and navigate to `http://localhost:3000/dev/[ANY_ROUTE]` (where `[ANY_ROUTE]` is any endpoint registered in your application).
 
 In the sections above, we've shown that using `webpack` and bundling your app can have significant impact on the overall bootstrap time.
-However, to make it work with our example, there's one additional configuration you must add in your `webpack.config.js` file. Generally,
-to make sure our `handler` function will be picked up, we must change the `output.libraryTarget` property to `commonjs2`.
+However, to make it work with our example, there are few additional configurations you must add in your `webpack.config.js` file. Generally,
```
Comment (jmcdo29) [inline content/faq/serverless.md:266]:
```suggestion
However, to make it work with our example, there are a few additional configurations you must add in your `webpack.config.js` file. Generally,
```

**Review by jmcdo29 — COMMENTED** [review]
Just a small change to help  the grammar, otherwise LGTM

**Review by jmcdo29 — APPROVED** [review]
LGTM

**Pull request merged**